src/index.html

```diff
@@ -11,6 +11,93 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+
+    <div class="container">
+      <div class="search search-big">
+        <form
+        action="https://www.w3schools.com/html/html_forms_attributes.asp"
+        class="searsh-form search-form-big"
+        method="post"
+        data-qa="big"
+        >
+          <input
+          type="text"
+          class="bar-input bar-input-big"
```

Comment:
checklist point 2
write all the styles for search_bar, but for .search_bar--small write only those styles that differ in design

Reply:
Thanks! I've deleted unnecessary indent and styles